src/commands/environments/destroy.ts

```diff
@@ -22,47 +23,72 @@ export default class EnvironmentDestroy extends Command {
   };
 
   static args = [{
-    name: 'name',
+    name: 'environment',
     description: 'Name of the environment to destroy',
     required: true,
+    parse: (value: string) => value.toLowerCase(),
+  }, {
+    name: 'account_name',
+    description: 'Account that the environment belongs to',
+    required: false
   }];
 
   async run() {
-    const {args, flags} = this.parse(EnvironmentDestroy);
+    const { args, flags } = this.parse(EnvironmentDestroy);
 
-    let answers: any = await inquirer.prompt([
-      {
-        type: 'autocomplete',
-        name: 'environment',
-        message: 'Select environment:',
-        source: async (_: any, input: string) => {
-          const params = { q: input };
-          const { data: environments } = await this.app.api.get('/environments', { params });
-          return environments.map((environment: any) => environment.name);
-        },
-        when: !args.environment,
```

Comment:
Why did we get rid of the autocomplete here?

Reply:
That's a required param, so it would never be used anyway